packages/safe-tools-client/app/components/schedule-collapse-panel/index.css

```diff
@@ -0,0 +1,136 @@
+.collapse-panel {
+  --collapse-panel-border-radius: 20px;
+  --collapse-panel-summary-height: 80px;
+  --collapse-panel-summary-anim-bottom-margin: 8px;
+
+  background: url('/images/backgrounds/schedule-panel.svg') no-repeat bottom/cover white;
+  border-radius: var(--collapse-panel-border-radius);
+  overflow: hidden;
+}
+
+.collapse-panel__summary {
```

Comment:
```
.collapse-panel__summary {list-style: none}
.collapse-panel__summary::-webkit-details-marker {display: none; }
```

this should reset and prevent the summary markers from showing on most browsers - they're currently showing on Safari.

Reply:
Thanks for checking that, I totally forgot to test on safari 🤦‍♀️